setup.py

```diff
@@ -1,36 +1,19 @@
 import os
-import sys
+from pathlib import Path
+from typing import List
 
 import setuptools
 
-try:
-    import builtins
-except ImportError:
-    import __builtin__ as builtins
+_PATH_ROOT = Path(os.path.dirname(__file__))
 
-PATH_ROOT = PATH_ROOT = os.path.dirname(__file__)
-builtins.__LIGHTLY_SETUP__ = True
 
-import lightly
```

Comment:
The `lightly` import was only used to read `lightly.__version__`. This is no longer necessary as we now read the value directly from the `lightly/__init__.py` file without imports. See `load_version` function for more info.